docs/releases/1.9.txt

```diff
@@ -118,6 +118,9 @@ Cache
 * ``django.core.cache.backends.base.BaseCache`` now has a ``get_or_set()``
   method.
 
+* ``django.views.decorators.cache.never_cache`` now sends more persuasive
+  headers to the browser to prevent it from caching (:ticket:`13008`).
```

Comment:
"... to prevent caching"
no need to include ticket reference in major release notes